Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -23,18 +23,25 @@ import PackageModel
 import SourceControl
 import _InternalTestSupport
 import Workspace
-import XCTest
+import Testing
 
-class APIDiffTestCase: CommandsBuildProviderTestCase {
-    override func setUpWithError() throws {
-        try XCTSkipIf(type(of: self) == APIDiffTestCase.self, "Skipping this test since it will be run in subclasses that will provide different build systems to test.")
+extension Trait where Self == Testing.ConditionTrait {
+    /// Skip test if the host operating system does not match the running OS.
```

Comment:
issue (possibly-blocking): this code comments appears to be a copy-paste error.  Can we add a more accurate comment, or remove it all together?